verification/verify.go

```diff
@@ -101,6 +107,9 @@ func (sv *SegmentVerifier) Verify(params *Params) (*Params, error) {
 	// TODO sig checking; extract from broadcast.go
 
 	if sv.policy.Verifier == nil {
+		if err := verifyPixelParams(params); err != ErrPixelsAbsent {
```

Comment:
The `ErrPixelsAbsent` is returned if offchain mode is used, right? Assuming we still need the onchain/offchain distinction, could `nil` be returned  for onchain from `verifyPixelParams`, then things simplify to:

```golang
if sv.policy.Verifier == nil {
  return nil, verifyPixelParams(params)
}
```

Reply:
done in d1eb9160b7957ec91343d28bd1a8a18bc2c63d68